.github/pull_request_template.md

```diff
@@ -1,23 +1,25 @@
-<!--
+TODO: Please describe your changes here.
+TODO: Delete the following lines but _keep and fill out_ (!) the checklist below.
+
 Thank you for your pull request!
```

Comment:
I think you might want a blank line after this one too.

Reply:
Added a line break, not a paragraph break.